README.md

```diff
@@ -9,4 +9,12 @@ For a list of examples please visit https://docs.edgexfoundry.org/1.2/examples/A
 ### Build Prerequisites
 
 Please see the [edgex-go README](https://github.com/edgexfoundry/edgex-go/blob/master/README.md).
+
+## Versioning
```

Comment:
Just a nit, the new `headings` are different size then that of `Build Prerequisites`. These should be consistent.